extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,72 +1,74 @@
-<div class="wds-global-navigation <?= isset( $model['fandom_overview'] ) ? '' : ' wds-search-is-always-visible'; ?>">
-	<div class="wds-global-navigation__content-bar">
-		<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'logo', [ 'model' => $model['logo'] ] ); ?>
-		<?php
-		if ( isset( $model['fandom_overview'] ) ):
-			foreach ( $model['fandom_overview']['links'] as $link ):
-		?>
-				<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkBranded', [ 'model' => $link ] ); ?>
-		<?php
-			endforeach;
-		endif;
-		?>
-		<?= $app->renderView(
-			'DesignSystemGlobalNavigationService',
-			'links',
-			[
-				'model' => $model['wikis'],
-				'type' => 'wikis-menu'
-			]
-		); ?>
-		<form class="wds-global-navigation__search" action="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['url'] ); ?>">
-			<div class="wds-global-navigation__search-input-wrapper">
-				<label class="wds-global-navigation__search-label">
-					<?= DesignSystemHelper::getSvg(
-						'wds-icons-magnifying-glass',
-						'wds-icon wds-icon-small wds-global-navigation__search-label-icon'
-					) ?>
-					<input class="wds-global-navigation__search-input"
-						name="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['param-name'] ); ?>"
-						data-active-placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-active'] ); ?>"
-						placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
-				</label>
-				<button class="wds-button wds-is-text wds-global-navigation__search-close" type="reset">
-					<?= DesignSystemHelper::getSvg(
-						'wds-icons-cross',
-						'wds-icon wds-icon-small wds-global-navigation__search-close-icon',
-						wfMessage( 'global-navigation-search-cancel' )->escaped()
-					) ?>
-				</button>
-			</div>
-			<button class="wds-button wds-global-navigation__search-submit">
-				<?= DesignSystemHelper::getSvg(
-					'wds-icons-arrow',
-					'wds-icon wds-icon-small wds-global-navigation__search-submit-icon'
-				) ?>
-			</button>
-		</form>
-		<?php if ( isset( $model['user'] ) ): ?>
+<div class="wds-global-navigation-wrapper">
+	<div id="globalNavigation" class="wds-global-navigation <?= isset( $model['fandom_overview'] ) ? '' : ' wds-search-is-always-visible'; ?>">
+		<div class="wds-global-navigation__content-bar">
+			<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'logo', [ 'model' => $model['logo'] ] ); ?>
+			<?php
+			if ( isset( $model['fandom_overview'] ) ):
+				foreach ( $model['fandom_overview']['links'] as $link ):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "links" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)